CIP-token-metadata-translations/README.md

```diff
@@ -0,0 +1,153 @@
+              <other languages and cultures>
+          }
+        },
+      },
+      "version": <version_id>,
+                  
+      <information about collection>
+      
+      "strings": {
+              "de-CH": {
+                 <localized information about collection in de-CH>
+              },
+              "it-IT": {
+                 <localized information about collection in it-IT>
+              },
+                    
+              <other languages and cultures>
+      }
+  }
+}
+
+```
+
+This metadata standard extension is backward compatible and it doesn't affect applications using the current standard. Dapps implementing the proposed extended standard can also default on the legacy values if the localized strings are not available on an asset.
+
+> **Note**
+> The metadata's size is a limitation that should be considered as it will eventually push the boundaries of Cardano's transaction limits and scalability. The translations under the "strings" keys can be stored off-chain on an IPFS server. Then, the localized texts will be accessible through an URL, similar to the "image" keys.
+
+### Code example to access localized properties (TypeScript)
+
+To access the localized strings from the fetched metadata for a native asset, we can simply access the JSON properties from the front end by using the user's selected culture:
+
+```
+
+const response = await fetch(`${<BASE_URL>}/policyId/${<policyId>}`);
+            
+const metadata = response.json();
+const policy = metadata["721"][<policy_id>];
+
+function getPolicyString(policy, key, culture="en") {
+    return policy["strings"][culture][key] 
+        ? policy["strings"][culture][key] 
+        : policy[key]; // default value (not localized)
+}
+
+function getAssetString(policy, asset, key, culture="en") {
+    return policy[asset]["strings"][culture][key] 
+        ? policy[asset]["strings"][culture][key] 
+        : policy[asset][key]; // default value (not localized)
+}
+
+console.log(`Default policy property: ${getPolicyString(policy, <policy_property>)}`);
+console.log(`Localized policy property: ${getPolicyString(policy, <policy_property>, "it-IT")}`);
+console.log(`Default asset name: ${getAssetString(policy, <asset_name>, "name")}`);
+console.log(`Localized asset name: ${getAssetString(policy, <asset_name>, "name", "de-CH")}`);
+         
+```
+     
+## Path to Active
+
+### Acceptance Criteria
+- NFT metadata standard extension covers all existing localization needs and use cases on web2.
+- Implementation is compliant with JSON conventions.
+- Implementation is compliant with the [[ISO-639]](https://www.iso.org/standard/4767.html) standard for language code, and the [[ISO-3166]](https://www.iso.org/standard/63545.html) standard for country/region code.
+- Access to localized strings is easy and logical from a coding perspective.
+            
+### Implementation Plan
+Add this new standard to all relevant documentation and references for web3 developers.

```

Comment:
```suggestion
Propose this method in documentation and references for web3 developers.

```
1. Here and elsewhere I would suggest avoiding the term "standard" since only adoption & longevity can characterise it as such.  Until that time comes, this is only a "method" or a "proposal".
2. All items in the Path to Active would work well as tickboxes; see the GitHub formatting and/or more recent CIPs to see how we've been doing these, which would suit your CIP's progression from `Proposed` to `Active` as they are ticked,